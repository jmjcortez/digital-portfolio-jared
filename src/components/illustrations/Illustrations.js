import React from 'react'

import './Illustrations.styles.css';

import doneCrying from '../../assets/images/done_crying.png';
import threeAm from '../../assets/images/3am_vibes.png';
import kamn from '../../assets/images/kamn.png';
import colours from '../../assets/images/colours.png';
import mentalTorture from '../../assets/images/mental_torture.png';
import handsDown from '../../assets/images/hands_down.png';
import iHope from '../../assets/images/i_hope_to_be_around.png';
import griselda from '../../assets/images/griselda.png';

const Illustrations = () => {
  return (
    <div className="gallery" data-testid='gallery'>
      <div className="piece-container">
        <img className="piece" src={doneCrying} alt="done_crying" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
      <div className="piece-container">
        <img className="piece" src={threeAm} alt="3am_vibes" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
      <div className="piece-container">
        <img className="piece" src={kamn} alt="kamn" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
      <div className="piece-container">
        <img className="piece" src={colours} alt="colours" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
      <div className="piece-container">
        <img className="piece" src={mentalTorture} alt="mental_torture" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
      <div className="piece-container">
        <img className="piece" src={handsDown} alt="hands_down" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
      <div className="piece-container">
        <img className="piece" src={iHope} alt="i_hope" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
      <div className="piece-container">
        <img className="piece" src={griselda} alt="griselda" />
        <div className="caption">
          <h1>(done) crying</h1>
          <p>reflection of anxiety in someone</p>
        </div>
      </div>
    </div>
  )
}

export default Illustrations