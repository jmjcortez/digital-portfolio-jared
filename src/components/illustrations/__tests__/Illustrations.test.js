import '@testing-library/jest-dom';

import { render, screen, cleanup } from '@testing-library/react';
import renderer from 'react-test-renderer';

import Illustrations from '../Illustrations';

afterEach(() => {
  cleanup();
});

test('should render Illustrations', () => {
  render(<Illustrations />);
  const illustrationElement = screen.getByTestId('gallery');
  expect(illustrationElement).toBeInTheDocument();
});

test('matches snapshot', () => {
  const tree = renderer.create(<Illustrations />);
  expect(tree).toMatchSnapshot();
});