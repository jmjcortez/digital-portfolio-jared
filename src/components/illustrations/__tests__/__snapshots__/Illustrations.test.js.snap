// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="gallery"
  data-testid="gallery"
>
  <div
    className="piece-container"
  >
    <img
      alt="done_crying"
      className="piece"
      src="done_crying.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
  <div
    className="piece-container"
  >
    <img
      alt="3am_vibes"
      className="piece"
      src="3am_vibes.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
  <div
    className="piece-container"
  >
    <img
      alt="kamn"
      className="piece"
      src="kamn.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
  <div
    className="piece-container"
  >
    <img
      alt="colours"
      className="piece"
      src="colours.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
  <div
    className="piece-container"
  >
    <img
      alt="mental_torture"
      className="piece"
      src="mental_torture.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
  <div
    className="piece-container"
  >
    <img
      alt="hands_down"
      className="piece"
      src="hands_down.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
  <div
    className="piece-container"
  >
    <img
      alt="i_hope"
      className="piece"
      src="i_hope_to_be_around.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
  <div
    className="piece-container"
  >
    <img
      alt="griselda"
      className="piece"
      src="griselda.png"
    />
    <div
      className="caption"
    >
      <h1>
        (done) crying
      </h1>
      <p>
        reflection of anxiety in someone
      </p>
    </div>
  </div>
</div>
`;
